zingolib/src/blaze/block_management_reorg_detection.rs

```diff
@@ -63,14 +63,13 @@ pub struct BlockManagementData {
     pub sync_status: Arc<RwLock<BatchSyncStatus>>,
 }
 
-pub const BATCHSIZE: u32 = 25;
 impl BlockManagementData {
     pub fn new(sync_status: Arc<RwLock<BatchSyncStatus>>) -> Self {
         Self {
             blocks_in_current_batch: Arc::new(RwLock::new(vec![])),
             existing_blocks: Arc::new(RwLock::new(vec![])),
             unverified_treestates: Arc::new(RwLock::new(vec![])),
-            batch_size: BATCHSIZE,
+            batch_size: zingoconfig::BATCH_SIZE as u32,
```

Comment:
Why is this a u32 and the other one a u64? Non-blocking.

Reply:
I don't know.